finders.go

```diff
@@ -19,7 +19,10 @@ var rLimit = regexp.MustCompile("(?i)(limit [0-9]+)$")
 //
 //	c.Find(&User{}, 1)
 func (c *Connection) Find(model interface{}, id interface{}) error {
-	return Q(c).Find(model, id)
+	q := Q(c)
```

Comment:
would it be worth making the `Q` method set those attributes for you, instead of doing it repeatedly?

Reply:
@larrymjordan @markbates is this the one thing holding this up?